app/js/controllers/service-tags.js

```diff
@@ -0,0 +1,14 @@
+import React from 'components/react';
+import ReactDOM from 'components/react-dom';
+import { Controller } from 'components/mvc';
+
+import ServiceTagsView from 'js/views/service-tags';
+
+export default class ServiceTagsController extends Controller {
+  main(id) {
+    ReactDOM.render(React.createElement(ServiceTagsView, {
+      id,
+      foxbox: this.foxbox,
+    }), this.mountNode);
+  }
+}
```

Comment:
Let's hold on this discussion for now until we find a better framework for the app.